labman/db/tests/test_composition.py

```diff
@@ -320,21 +340,45 @@ def test_attributes(self):
 
     def test_get_next_combos(self):
         tester = ShotgunPrimerSet(1)
-        self.assertEqual(tester.current_combo_index, 384)
+        self.assertEqual(tester.current_combo_index, 380)
```

Comment:
It's not clear to me why this value changed, same with the one on line 311, would it be possible to provide some context?